src/rtFileDownloader.cpp

```diff
@@ -366,6 +366,29 @@ bool rtFileDownloadRequest::isDataCached()
   return mIsDataInCache;
 }
 
+void rtFileDownloadRequest::setReadCachedDataInChunks(bool val)
+{
+  mReadCacheDataInChunks = val;
+}
+
+bool rtFileDownloadRequest::readCachedDataInChunksEnabled()
+{
+  return mReadCacheDataInChunks;
+}
```

Comment:
Let's rename setReadCachedDataInChunks() and readCachedDataInChunksEnabled() APIs to setDeferCacheRead() and deferCacheRead().